provisioning/service/src/Contract/ContractApiHttp.cs

```diff
@@ -122,11 +122,19 @@ internal class ContractApiHttp : IContractApiHttp
                             $"The response message was null when executing operation {httpMethod}.");
                     }
 
+#if NET5_0
```

Comment:
Another example of a lot of DRY violations here, that wouldn't be as bad if you used the #if NET5_0 differently.

Reply:
sure, I wasn't sure how the inline if-else params within a method call would be like, but I'll definitely try it out.

Reply:
I think if the items that are different between frameworks still return the same type a private method would be helpful so you can isolate the NET5_0 compiler statements, or even a local function that does the same thing